main.go

```diff
@@ -106,32 +106,35 @@ func main() {
 		os.Exit(1)
 	}
 
-	clusterID, err := getClusterID(mgr.GetAPIReader())
+	clusterVersion, err := getClusterVersion(mgr.GetAPIReader())
```

Comment:
This assumes that the cluster version stays the same for the lifetime of a CLO instance, but what
if the cluster is upgraded under a running CLO?

Reply:
then the operator will be redeployed.  I don't see how the cluster could be upgraded and the operator not get restarted